client/src/pages/club/match/CreateMatch.tsx

```diff
@@ -37,91 +47,179 @@ const MapView = styled.div`
-  }
-
-  const mapViewModalHandler = ():void => {
-    if(!placeValue) return;
-    setIsOpenMapView(!isOpenMapView);
-  }
   const [teamList, setTeamList] = useState([
     {
       id: 0,
       members: []
     }
   ]);
-  const memberList:string[] = ["박대운", "우제훈", "김은택", "김아애", "문채리", "전규언"];
+
+  //match type을 BE table 구조를 보고 정확하게 정하지 못한상태여서 any타입으로 임시지정.
+  const [matchResult, setMatchResult] = useState<any>([]);
+  
+  const candidates: string[] = ['박대운', '우제훈', '김은택', '김아애', '문채리', '전규언'];
+
+  const [isOpenAddMember, setIsOpenAddMember] = useState(false);
+  const [addButtonIndex, setAddButtonIndex] = useState(0)
+
+  const mapSettingModalHandler = (): void => {
+    setIsOpenMapSetting(!isOpenMapSetting);
+  };
+
+  const mapViewModalHandler = (): void => {
+    if (!placeValue) return;
+    setIsOpenMapView(!isOpenMapView);
+  };
+
   return (
     <S_Container>
-      <div>경기 등록</div>
+      <S_Title>경기 등록</S_Title>
+      <br />
       <div>
-        <div>날짜/시간 선택</div>
-        <input type='date' />
-        <input type='time' />
+        <S_Label>날짜/시간 선택</S_Label>
+        <S_Input type='date' />
+        <S_Input type='time' />
       </div>
+      <br />
       <div>
-        <div>장소</div>
-        <input type='text' value={placeValue?.place_name} readOnly/>
+        <S_Label>장소</S_Label>
+        <S_Input type='text' value={placeValue?.place_name} readOnly />
         <button onClick={mapSettingModalHandler}>지도설정</button>
         <button onClick={mapViewModalHandler}>지도보기</button>
-        {
-          isOpenMapSetting ? 
-          <MapBackdrop onClick={mapSettingModalHandler}>
-            <MapView onClick={(e) => e.stopPropagation()}>
-              <KakaoMapSearch mapSettingModalHandler={mapSettingModalHandler} setPlaceValue={setPlaceValue}/>
-            </MapView>
-          </MapBackdrop>
-          : null
-        }
-        {
-          isOpenMapView ?
-          <MapBackdrop onClick={mapViewModalHandler}>
-            <MapView onClick={(e) => e.stopPropagation()}>
-              <KakaoMapView place={placeValue}/>
-            </MapView>
-          </MapBackdrop>
-          : null
-        }
-        <div>참석자</div>
-        <div>경기를 등록하면 경기정보 페이지에서 참석/불참 여부를 받을 수 있어요.</div>
-        <div>참석을 선택한 멤버는 자동으로 등록됩니다. </div>
-        <button>+</button>
+        {isOpenMapSetting ? (
+          <S_MapBackdrop onClick={mapSettingModalHandler}>
+            <S_MapView onClick={(e) => e.stopPropagation()}>
+              <KakaoMapSearch
+                mapSettingModalHandler={mapSettingModalHandler}
+                setPlaceValue={setPlaceValue}
+              />
+            </S_MapView>
+          </S_MapBackdrop>
+        ) : null}
+        {isOpenMapView ? (
+          <S_MapBackdrop onClick={mapViewModalHandler}>
+            <S_MapView onClick={(e) => e.stopPropagation()}>
+              <KakaoMapView place={placeValue} />
+            </S_MapView>
+          </S_MapBackdrop>
+        ) : null}
```

Comment:
요렇게 사용하시면 좀 더 편할 것 같아요~
```suggestion
        {isOpenMapView && (
          <S_MapBackdrop onClick={mapViewModalHandler}>
            <S_MapView onClick={(e) => e.stopPropagation()}>
              <KakaoMapView place={placeValue} />
            </S_MapView>
          </S_MapBackdrop>
        )}
```

Reply:
이 부분 관련한 react 공식 문서 링크 저장해둔 게 있어서 공유합니다.
https://ko.reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator